xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2156,6 +2156,7 @@ void CVideoPlayer::CheckContinuity(CCurrentStream& current, DemuxPacket* pPacket
        fabs(this_dts - that_dts) < DVD_MSEC_TO_TIME(1000))
     {
       m_offset_pts += correction;
+      m_VideoPlayerVideo->SetOffsetPts(m_offset_pts);
```

Comment:
> CDVDDemuxSPU::AddData gets the corrected pts already

Aha... I think I found something.
iPTSStartTime of the "invisible" overlay is still 0.00000000000000000, the iPTSStopTime is 2533274790395904.0 and the pts_org -94802158466.666672
packet->pts seems to be -94789070422.222229
But: because https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDDemuxSPU.cpp#L107
pSPUData->pts remains 0.000000... thus the corrected PTS doesn't get assigned.